articles/videos/get-started/04_01-authenticate-how-it-works.md

```diff
@@ -0,0 +1,159 @@
+---
+description: How user authentication works and various ways to accomplish it with Auth0.
+classes: video-page
+---
+# Authenticate: How It Works
+
+Learn about the difference between authentication, authorization, and access control. Understand when and why you might use each type of authentication method: First factors, second factors, and multi-factor. Learn about the OpenID Connect authentication protocol.   
```

Comment:
Don't capitalize "First". For consistency with other pages,  add "(OIDC)" after "OpenID Connect".